docs-src/design-decisions.md

```diff
@@ -0,0 +1,44 @@
+# Design Decisions
+
+Throughout the design of these APIs some significant design decisions have been
+made. This provides some background on the factors considered when making these
+decisions.
+
+## GatewayClass Scope
+
+### Problem
+There were a number of compelling reasons to make GatewayClass a
+namespace-scoped resource. 
+
+### Advantages of a Namespace-Scoped GatewayClass
+* Less potential for naming conflicts among different implementations.
+* Enables using Service API on a management cluster designed to provision and
```

Comment:
comment got lost, recreating from memory.

I object to "management cluster" as a term of art without a definition.  I don't think there's any established meaning or semantic expectations for it.

Reply:
https://cluster-api.sigs.k8s.io/reference/glossary.html#management-cluster

Reply:
Good call. I've added a link to the management cluster definition above.